pkgs/by-name/cy/cyclonedx-cli/package.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildDotnetModule
+, fetchFromGitHub
+}:
+
+buildDotnetModule rec {
+  pname = "cyclonedx-cli";
+  version = "0.25.0";
+
+  src = fetchFromGitHub {
+    owner = "CycloneDX";
+    repo = "${pname}";
+    rev = "v${version}";
+    hash = "sha256-kAMSdUMr/NhsbMBViFJQlzgUNnxWgi/CLb3CW9OpWFo=";
+  };
+
+  nugetDeps = ./deps.nix;
+
+  meta = with lib; {
+    description = "CycloneDX CLI tool for SBOM analysis, merging, diffs and format conversions. ";
```

```suggestion
    description = "CycloneDX CLI tool for SBOM analysis, merging, diffs and format conversions";
```

No period at the end please.